pkgs/development/python-modules/pytorch/default.nix

```diff
@@ -36,14 +39,6 @@ in buildPythonPackage rec {
     sha256 = "076cpbig4sywn9vv674c0xdg832sdrd5pk1d0725pjkm436kpvlm";
```

Comment:
Is this SHA correct? Looks the same to me as 1.0.0?

Reply:
ah! Yes, I'm primarily working off of https://github.com/stites/libtorch-nix and missed this when transferring over details. Good catch -- I'm a bit surprised this slipped through since I was building off of my fork (using the command from the PR message).